modules/airgap/setup/get_artifacts.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+## Uncomment the following lines to enable debug mode
+# set -x
+# echo "$@"
+
+# Usage: ./get_artifacts.sh k3s v1.27.5+k3s1
+# Usage: ./get_artifacts.sh rke2 v1.27.5+rke2 amd64 "flags" tar.gz
+
+product=$1
+version=$2
+arch=$3
+flags=${4}
+tarball_type=$5
+prodbin=$product
+
+
+check_arch(){
+  if [[ -n "$arch" ]] && [[ "$arch" = *"arm"* ]]; then
+    if [[ "$product" == "k3s" ]]; then
+      prodbin="k3s-arm64"
+    else
+      arch="arm64"
+    fi
+  else
+    arch="amd64"
+  fi
+}
+
+get_assets() {
+  echo "Downloading $product dependencies..."
+  if [[ "$product" == "k3s" ]]; then
+    url="https://github.com/k3s-io/k3s/releases/download/$version"
+    wget "$url"/k3s-images.txt
+    wget -O k3s-install.sh https://get.k3s.io/
+    wget -O k3s "$url"/"$prodbin"
+    if [ -n "$tarball_type" ]; then
+      wget "$url"/k3s-airgap-images-"$arch"."$tarball_type"
+    fi
+  elif [[ "$product" == "rke2" ]]; then
+    url="https://github.com/rancher/rke2/releases/download/$version"
+    wget "$url"/sha256sum-"$arch".txt
+    wget "$url"/rke2-images.linux-"$arch".txt
+    wget "$url"/rke2.linux-"$arch".tar.gz
```

Comment:
Can we add a check to confirm any image is not missing? ref: https://github.com/rancher/rke2/issues/6438

Reply:
You mean adding check that these files are downloaded?

Reply:
Will add it on next phase...

Reply:
I'm good with that.